tests/ExamplesTest.php

```diff
@@ -31,15 +36,16 @@ public function testExample($example, $output)
     public function getExamples()
     {
         return [
-            ['misc', 'misc.json'],
-            ['openapi-spec', 'openapi-spec.json'],
-            ['petstore.swagger.io', 'petstore.swagger.io.json'],
-            ['petstore-3.0', 'petstore-3.0.json'],
-            ['swagger-spec/petstore', 'petstore.json'],
-            ['swagger-spec/petstore-simple', 'petstore-simple.json'],
-            ['swagger-spec/petstore-with-external-docs', 'petstore-with-external-docs.json'],
-            ['using-refs', 'using-refs.json'],
-            ['example-object', 'example-object.json'],
+//            ['misc', 'misc.json', []],
```

Comment:
Is it possible to add a custom command line option to phpunit? Something like:
`phpunit --swagger-example using-traits` 
Right now the other examples are excluded from running in travisCI and easy to forget to re-enable when te PR is ready.

Reply:
Yeah, that was an oversight from doing manual testing. I'll restore that once we are happy with the changes.
I guess I could user `--filter using-traits` but then it gets complicated when trying to debug in the IDE...